src/registrar/tests/test_views_portfolio.py

```diff
@@ -837,3 +837,106 @@ def test_portfolio_cache_updates_when_flag_disabled_while_logged_in(self):
             response = self.client.get(reverse("home"))
             self.assertIsNone(self.client.session.get("portfolio"))
             self.assertNotContains(response, "Hotel California")
+
+    @less_console_noise_decorator
+    @override_flag("organization_feature", active=True)
+    @override_flag("organization_requests", active=True)
+    def test_delete_domain_request_as_org_user_with_permission_and_deletable_status(self):
```

Comment:
nit but are we open to shortening the length of this test name? eg test_org_user_can_delete_own_domain_request?  (which is also a fairly long test name so feel free to go at your discretion)